code/modules/power/floodlight.dm

```diff
@@ -13,6 +13,44 @@
 	density = TRUE
 	var/state = FLOODLIGHT_NEEDS_WIRES
 
+/obj/structure/floodlight_frame/screwdriver_act(mob/living/user, obj/item/O)
+	. = ..()
+	if(state == FLOODLIGHT_NEEDS_SECURING)
+		to_chat(user, span_notice("You fasten the wiring and electronics in [src]."))
+		name = "secured [name]"
+		desc = "A bare metal frame that looks like a floodlight. Requires a light tube to complete."
+		icon_state = "floodlight_c3"
+		state = FLOODLIGHT_NEEDS_LIGHTS
+		return TRUE
+	else if(state == FLOODLIGHT_NEEDS_LIGHTS)
+		to_chat(user, span_notice("You unfasten the wiring and electronics in [src]."))
+		name = "wired [name]"
```

Comment:
Isn't this going to end up being "wired secured floodlight"? Feels weird

Reply:
Like surely you'll be able to stack these. That is weird.

Reply:
So we make them dense at each step to avoid stacking?I'll change the naming that's fine